lib/components/narrative/tabbed-itineraries.js

```diff
@@ -130,6 +131,9 @@ class TabButton extends Component {
     // TODO: support non-USD
     const transitFare = (transitFares?.[defaultFareKey] || transitFares.regular)?.transitFare || 0
     const minTotalFare = minTNCFare * 100 + transitFare
+    const formattedStartTime = itinerary.startTime + timezoneOffset
+    const formattedEndTime = itinerary.endTime + timezoneOffset
```

Comment:
Rename, these are just time stamps now.

Reply:
Renamed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/b4b757b5e0da96045794b7a642d7271e79218c10